rskj-core/src/main/resources/reference.conf

```diff
@@ -84,6 +84,7 @@ database {
     reset = false
     unitrie-migration {
         minimum-height: 0
+        missing-keys-url: "https://github.com/rsksmart/unitrie-migration/blob/master/migration-extras?raw=true"
```

Comment:
Did you hear from QA about the key override issue?

Reply:
finally it wasn't an override issue, it was just me retrieving the wrong key